src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml

```diff
@@ -24,54 +24,54 @@ limitations under the License.
     <client init="false" tick="false">true</client>
     <server init="false" tick="false">true</server>
     <description>This cluster provides an interface for controlling the current Channel on a device.</description>
-    <attribute side="server" code="0x0000" define="CHANNEL_LIST"            type="ARRAY" entryType="ChannelInfo" length="254"                    writable="false" optional="true">ChannelList</attribute>
-    <attribute side="server" code="0x0001" define="CHANNEL_LINEUP"          type="LineupInfo"                    default="0x0" isNullable="true" writable="false" optional="true">Lineup</attribute>
-    <attribute side="server" code="0x0002" define="CHANNEL_CURRENT_CHANNEL" type="ChannelInfo"                   default="0x0" isNullable="true" writable="false" optional="true">CurrentChannel</attribute>
+    <attribute side="server" code="0x0000" define="CHANNEL_LIST"            type="ARRAY" entryType="ChannelInfoStruct" length="254"                    writable="false" optional="true">ChannelList</attribute>
+    <attribute side="server" code="0x0001" define="CHANNEL_LINEUP"          type="LineupInfoStruct"                    default="0x0" isNullable="true" writable="false" optional="true">Lineup</attribute>
+    <attribute side="server" code="0x0002" define="CHANNEL_CURRENT_CHANNEL" type="ChannelInfoStruct"                   default="0x0" isNullable="true" writable="false" optional="true">CurrentChannel</attribute>
 
     <command source="client" code="0x00" name="ChangeChannel" response="ChangeChannelResponse" optional="true">
       <description>Change the channel on the media player to the channel case-insensitive exact matching the value passed as an argument. </description>
-      <arg name="match" type="CHAR_STRING"/>
+      <arg name="Match" type="CHAR_STRING"/>
     </command>
 
     <command source="client" code="0x02" name="ChangeChannelByNumber" optional="false">
       <description>Change the channel on the media plaeyer to the channel with the given Number in the ChannelList attribute.</description>
-      <arg name="majorNumber" type="INT16U"/>
-      <arg name="minorNumber" type="INT16U"/>
+      <arg name="MajorNumber" type="INT16U"/>
```

Comment:
oddly enough, my copy of the spec has these starting with lowercase (even though that seems inconsistent with all other structs I have seen)